src/constructs/StaticWebsite.ts

```diff
@@ -278,6 +272,34 @@ export class StaticWebsite extends AwsConstruct {
         return this.provider.getStackOutput(this.distributionIdOutput);
     }
 
+    private errorResponse(): ErrorResponse {
+        // Custom error page
+        if (this.configuration.errorPage !== undefined) {
+            let errorPath = this.configuration.errorPage;
+            if (!errorPath.startsWith("/")) {
+                errorPath = `/${errorPath}`;
```

Comment:
If errorPath begins with a dot like `./error.html` then it would be invalid.

I think this case should be covered.

Reply:
Good point, I have added a check for that.